openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -211,6 +212,8 @@ def _declare_options(self):
                              desc='Name of optimizers to use')
         self.options.declare('title', default='Optimization using pyOpt_sparse',
                              desc='Title of this optimization run')
+        self.options.declare('print_opt_prob', types=bool, default=True,
```

Comment:
I would prefer if this had a default of `False`.

We're in the process of trying to move away from having so much standard output, and moving towards reports instead.  In fact, one of our team members is working on an optimization report that will summarize everything about the optimization problem.

Sometimes instant feedback via stdout is good, but I think the user should have to ask for it rather than enabling it by default.

Reply:
@robfalck Thanks for the feedback. I have changed it to default false and pushed it to the PR.